spec/features/versions_spec.rb

```diff
@@ -218,12 +218,21 @@
   scenario 'display changes in splashpages', feature: true, versioning: true, js: true do
     visit admin_conference_splashpage_path(conference.short_title)
     click_link 'Create Splashpage'
+    uncheck('Display tracks')
```

Comment:
@Ana06 that does not pass the test I think for that we have to also update the expect(page).to